dev/com.ibm.ws.security.appbnd_test/test/com/ibm/ws/security/appbnd/internal/authorization/AppBndAuthorizationTableServiceTest.java

```diff
@@ -0,0 +1,1127 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
```

Comment:
Is this date correct? The file above says 2017, 2020 so I would have expected the test classes to be the same?